test/check/check.go

```diff
@@ -509,9 +509,20 @@ func checkConfigCondition(gvk schema.GroupVersionKind, conditionType ConditionTy
 	return fmt.Errorf("condition %q has not been found in the config", conditionType)
 }
 
+func gatherClusterInfo() string {
+	podsStatus := cnaoPodsStatus()
+	describeAll := describeAll()
```

Comment:
the name `All` is now not accurate' since its not All,
wdyt?

maybe put the logic in the describeAll? it already return two strings that at the end are joined
is there any reason to keep it separate ?

Reply:
not sure what you mean there. 
`describeAll()` does exactly that `kubectl describe all -n <cnao-ns>`
similarly, `cnaoPodsStatus()` does `kubectl get pods -n <cnao-ns>`, but it's not describing anything, hence the separation

Reply:
ah ok than all good , just the All word mislead me
thanks